internal/commands/user/inputs.go

```diff
@@ -47,3 +57,56 @@ func displayUserData(apt realm.AuthProviderType, user realm.User) string {
 	}
 	return ""
 }
+
+func (i multiUserInputs) filter() realm.UserFilter {
+	return realm.UserFilter{
+		IDs:       i.Users,
+		State:     i.State,
+		Pending:   i.Pending,
+		Providers: realm.NewAuthProviderTypes(i.ProviderTypes...),
+	}
+}
+
+func (i multiUserInputs) resolveUsers(realmClient realm.Client, groupID, appID string) ([]realm.User, error) {
+	foundUsers, findErr := realmClient.FindUsers(groupID, appID, i.filter())
```

Comment:
[nit] consistent prefix

Reply:
changed to `foundUsers, err` since I liked the tense of `found` here